src/test/node/items/controllers.js

```diff
@@ -157,8 +157,9 @@ test('modify an item in database', t => {
   });
 });
 test('delete an item from database', t => {
+  const key = `item-${uuid()}`;
   const ops = [
-    { type: 'put', key: 'item1', value: itemRedSelo }
+    { type: 'put', key: key, value: itemRedSelo }
```

Comment:
you can just type {key}
[https://github.com/airbnb/javascript#es6-object-concise](https://github.com/airbnb/javascript#es6-object-concise)